caveclient/chunkedgraph.py

```diff
@@ -689,14 +807,41 @@ def remesh_level2_chunks(self, chunk_ids):
         r.raise_for_status()
 
     def get_operation_details(self, operation_ids: Iterable[int]):
-        """get the details of a list of operations
+        """Get the details of a list of operations.
 
-        Args:
-            operation_ids (Iterable[int]): list of operation IDss
+        Parameters
+        ----------
+        operation_ids: Iterable of int
+            List/array of operation IDs.
 
-        Returns:
-            dict: a dict of dictss of operation info, keys are operationids
-            values are a dictionary of operation info for the operation
+        Returns
+        -------
+        dict of str to dict
+            A dict of dicts of operation info, keys are operation IDs (as strings),
+            values are a dictionary of operation info for the operation. These
+            dictionaries contain the following keys:
+
+            "added_edges": list of list of int
+                List of edges added by this operation. Each edge is a list of two node
```

Comment:
i would save supervoxel_ids in this case

Reply:
done!